lib/ace/virtual_renderer_test.js

```diff
@@ -183,6 +184,22 @@ module.exports = {
         assert.ok(editor.session.lineWidgets[1]);
     },
     
+    "test wraped text rendering": function() {
```

Comment:
wrapped

Reply:
"test" is a bit unnecessary here..

Reply:
I agree, but this is the convention used in all tests